src/main/java/org/inferred/freebuilder/EqualsAndHashCode.java

```diff
@@ -0,0 +1,23 @@
+package org.inferred.freebuilder;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Generation options for {@link Object#equals(Object)} and {@link Object#hashCode()}.
+ */
+@Target(value = {})
+@Retention(RetentionPolicy.SOURCE)
+public @interface EqualsAndHashCode {
```

Comment:
I kept a top level placeholder that has no `@Target` so it's unusable. I was following semantics that `Lombok ` has used.

Reply:
A possible use case is that you could have annotation properties here, such as `include` or `exclude`. Although I'm not a huge fan of having something weakly typed, and I think `Lombok` gets away with it through their plugin which should tell Intellij or eclipse that a value is correct or incorrect.

Furthermore `Lombok` provides a lot of functionality that while great for someone who wants a lot of configurability, could be seen as counter to FreeBuilder more opinionated approach. Neither is necessarily wrong to me, but I want this feature to be consistent with the rest of FreeBuilder

For more reading on lombok, https://projectlombok.org/features/EqualsAndHashCode